Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -2947,6 +2949,7 @@ void RenderLayer::resize(const PlatformMouseEvent& evt, const LayoutSize& oldOff
         styledElement->setInlineStyleProperty(CSSPropertyHeight, roundToInt(baseHeight + difference.height()), CSSUnitType::CSS_PX);
     }
 
+    mutationScope.enqueueMutationRecord();
```

Comment:
You should call this in the branches that invoke `setInlineStyleProperty()` (calling it multiple times is fine) to avoid unnecessary work.